snovault/schema_utils.py

```diff
@@ -41,8 +39,10 @@ def server_default(func):
 
 
 class NoRemoteResolver(RefResolver):
+    """ This has been modified to allow remote resolution that does not involve an http, https or ftp url
```

Comment:
I don't think this is good form for a doc string, which should say what something does, not say how it's changed from something else, at least not without naming the something else.

```suggestion
    """
    A refinement of RefResolver that disallows http, https or ftp urls.
```